staff/ana-sanjuan/cv/styles.css

```diff
@@ -0,0 +1,116 @@
+@import url('https://fonts.googleapis.com/css?family=Crimson+Text');
+
+body {
+    box-sizing: border-box;
+    margin: 5%;
+    background-color:rgb(210, 222, 230);
+    font-family: 'Crimson Text', serif;
+    color:#212121;
+}
+
+.section {
+    margin: 4% 2%;
+    box-shadow: 10px 10px 5px 2px #0097A7;
+    padding: 3%;
+    background-color: #00BCD4;
+    border-radius: 10px;
+}
+
+header {
+    text-align: right;
+}
+
+.introduction {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    margin-left:auto;
```

Comment:
margin-left: auto;
margin-right: auto;

try it with  margin: 0 auto;

Reply:
good tip! thanks man! :)